Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams_test.py

```diff
@@ -634,6 +634,52 @@ def test_send_message(mocker, requests_mock):
     assert len(results) == 1
     assert results[0] == 'Message was sent successfully.'
 
+    # verify message is sent properly given email with uppercase letters to send to
```

Comment:
Split the function into multiple unit-test for better readability and maintainability